### PR TITLE
Added symfony 3.x version + CS fixes

### PR DESCRIPTION
**Pull request by andreybolonin**

### CODE REVIEW

**Comment by andrey1s** [comment]
also need update factory service, for example `Resources/config/reply_message.xml` and other

**Comment by andreybolonin** [comment]
What about minimum 3.x symfony versions?

File: Form/Type/ReplyMessageFormType.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public function getName()
         return 'clickatell_reply_message';
     }
 
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function setDefaultOptions(OptionsResolver $resolver)
```
Comment (andrey1s) [inline Form/Type/ReplyMessageFormType.php]: `setDefaultOptions` wrong parameter type
symfony 2.8 and 3.0 use configureOptions

**Comment by andrey1s** [comment]
we can use symfony ~2.8 and ~3.0

**Comment by andreybolonin** [comment]
Ok

**Pull request merged**